README.md

```diff
@@ -1,8 +1,7 @@
 # YaST - rewrite of the storage library
 
 [![Build Status](https://travis-ci.org/yast/yast-storage-ng.svg?branch=master)](https://travis-ci.org/yast/yast-storage-ng)
-[![Coverage
-Status](https://coveralls.io/repos/github/yast/yast-storage-ng/badge.svg?branch=master)](https://coveralls.io/github/yast/yast-storage-ng?branch=master)
+[![Coverage Status](https://img.shields.io/coveralls/yast/yast-storage-ng.svg)](https://coveralls.io/github/yast/yast-storage-ng?branch=master)
```

Comment:
The link should rather point to https://img.shields.io/coveralls/yast/yast-storage-ng/master.svg so it's obvious that the coverage is for the *master* branch.

(This will also make it easier to change when we split *master* to *SP2*.)